src/content/architecture/globalnet/_index.en.md

```diff
@@ -0,0 +1,64 @@
++++
+title = "Globalnet Controller"
+date = 2020-02-19T21:08:37+01:00
+weight = 5
++++
+## Introduction
+
+Submariner is a tool built to connect overlay networks of different
+Kubernetes clusters. These clusters can be on different public clouds
+or on-premise. An important use case for Submariner is to connect disparate
+independent clusters into a single cohesive multi-cluster. 
+
+However, one key limitation of submariner's current design is that it doesn't
+support overlapping CIDRs (ServiceCIDR and ClusterCIDR) across these clusters.
+Each cluster must use distinct CIDRs that don't conflict or overlap with any
+other cluster that is going to be part of cluster fleet.
```

Comment:
The wording here sounds more like a proposal to handle overlapping CIDRs but Globalnet has already landed. Perhaps change accordingly to:

 _However, by default, a limitation of submariner is that it doesn't handle
overlapping CIDRs (ServiceCIDR and ClusterCIDR) across these clusters.
Each cluster must use distinct CIDRs that don't conflict or overlap with any
other cluster that is going to be part of cluster fleet._

Reply:
Right @tpantelis. This PR basically moves the existing content from Readme to the website. Your point is valid though, I'll update the PR.